amplipi/updater/static/index.html

```diff
@@ -199,49 +199,49 @@ <h1><span class="text-white">Ampli</span><span class="text-danger">Pi</span></h1
               </div>
 
               <!-- Password -->
-              <div id="set-password column-item">
-                <h6 class="hover-label">
-                  Set Password
-                  <span class="hover-tooltip">
-                    This form lets you set a web interface password, which will make it so that you must input the same password to access the system's controls. Leaving the fields empty will remove password protection. Note that this only affects the web interface, not the system password, and will log out any active sessions.
-                  </span>
+              <div style="background-color: inherit; box-shadow: none; width: 100%;" class="card set-password column-item">
+                <h6 class="card-header">
+                  Set Password:
                 </h6>
-                <div class="set-password form-group">
+                <div class="set-password form-group indent">
+                  <div class="description-color">
+                    Set Password allows you set to a web interface password, which will make it so that you must input the same password to access the app
```

Comment:
I think that the password field deserves a better description, I prefer the old one to the new one, however I know it's pretty long, try to condense it a bit.